bookkeeper-server/src/main/java/org/apache/bookkeeper/util/collections/ConcurrentLongLongPairHashMap.java

```diff
@@ -75,26 +133,21 @@
         boolean test(long key1, long key2, long value1, long value2);
     }
 
-    public ConcurrentLongLongPairHashMap() {
-        this(DefaultExpectedItems);
-    }
-
-    public ConcurrentLongLongPairHashMap(int expectedItems) {
-        this(expectedItems, DefaultConcurrencyLevel);
-    }
-
-    public ConcurrentLongLongPairHashMap(int expectedItems, int concurrencyLevel) {
+    private ConcurrentLongLongPairHashMap(int expectedItems, int concurrencyLevel,
```

Comment:
I would keep and Deprecate all the public constructors.
Otherwise this is a breaking change and we cannot port it to older releases

Reply:
OK

Reply:
Fixed.
PTAL,thanks!